client/my-sites/stats/stats-purchase/stats-purchase-checkout-redirect.tsx

```diff
@@ -86,7 +88,8 @@ const getRedirectUrl = ( {
 	redirectUri?: string;
 	siteSlug: string;
 } ) => {
-	const isStartedFromJetpackSite = from.startsWith( 'jetpack' );
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+	const isStartedFromJetpackSite = from.startsWith( 'jetpack' ) || isOdysseyStats;
```

Comment:
```suggestion
	const isStartedFromJetpackSite = isOdysseyStats;
```
Could we just check `isOdysseyStats` since the start of the query string `from` is basically related to `isOdysseyStats`?

Reply:
Same reason as above. 😄